terraform/modules/active_directory/security_groups.tf

```diff
@@ -18,11 +18,14 @@ resource "aws_security_group" "ad_management_server" {
     to_port   = 0
     protocol  = "-1"
 
+    # tfsec:ignore:aws-vpc-no-public-egress-sgr
     cidr_blocks = ["0.0.0.0/0"]
+    description = "Allow all egress"
   }
 }
 
 resource "aws_security_group_rule" "domain_controllers_to_ca_1" {
+  description       = "Directory Services to CA port 135" # TODO:tfsec what is this for, NTP?
```

Comment:
Either for remote group policy updates or requesting a certificate
Not super sure but it was in AWS's guide and LDAPS didn't work without it

Reply:
Cool, I've left it vague for now